manifests/l/LuculentSystems/AltA2DP/1.2.0/LuculentSystems.AltA2DP.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=NVS0.CRLF.7-2-10.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: LuculentSystems.AltA2DP
+PackageVersion: 1.2.0
```

```suggestion
PackageVersion: 1.2.0.1
```
This must match Control Panel.

You also need to change `manifests/l/LuculentSystems/AltA2DP/1.2.0` to `manifests/l/LuculentSystems/AltA2DP/1.2.0.1`

---

Could you clarify where that requirement comes from?